tests/System.Buffers.Primitives.Tests/EqualityTests.cs

```diff
@@ -17,8 +17,7 @@ public void SpansMustNotBeBoxed()
 
             try
             {
-                span.Equals((object)span);
```

Comment:
Why do we even have these span tests in here? These same tests (or at least most of them) exist in corefx as well.